src/Query/QueryBuilder.php

```diff
@@ -1625,4 +1635,37 @@ public function disableResultCache(): self
 
         return $this;
     }
+
+    public function addComment(string $comment): self
+    {
+        $sanitizedComment = $this->sanitizeComment($comment);
+        if ($sanitizedComment === '') {
+            return $this;
+        }
+
+        $this->comments[] = $sanitizedComment;
+
+        $this->sql = null;
+
+        return $this;
+    }
+
+    private function getComments(): string
+    {
+        $comments = '';
+        foreach ($this->comments as $comment) {
+            $comments .= sprintf('/* %s */ ', $comment);
+        }
+
+        return $comments;
+    }
+
+    private function sanitizeComment(string $comment): string
+    {
+        $comment = str_replace(['*/', '/*'], '', $comment);
+        $comment = preg_replace('/[[:cntrl:]]+/u', ' ', $comment) ?? $comment;
+        $comment = preg_replace('/\s+/u', ' ', $comment) ?? $comment;
```

Comment:
Why should we normalize whitespaces here? This would essentially forbid multi-line comments, wouldn't it?

Reply:
Well, the query builder is an abstraction for SQL. Comments are a part of the language, so adding an abstraction for that is fine.

However, I don't see why an SQL abstraction should know about how to preprocess strings for any of those tools you've mentioned.

Please remove the whitespace normalization and add multiline comments to the tests.